cpp/oneapi/dal/test/engine/fixtures.hpp

```diff
@@ -230,11 +241,65 @@ class crtp_algo_fixture : public float_algo_fixture<std::tuple_element_t<0, Test
 
         return this->merge_compute_result(results);
     }
+};
 
-private:
-    Derived& derived() {
-        return *(static_cast<Derived*>(this));
+#ifdef ONEDAL_DATA_PARALLEL
+
+struct ccl_backend {};
+struct mpi_backend {};
+
+template <typename TestType, typename Derived>
+class crtp_par_algo_fixture : public crtp_base_algo_fixture<TestType, Derived> {
```

Comment:
It would be better to rename 'crtp_par_algo_fixture' to 'crtp_spmd_backend_algo_fixture' to better reflect it's purpose.

Reply:
Done